cdk/bin/cdk.ts

```diff
@@ -1,14 +1,24 @@
 import 'source-map-support/register';
-import { App } from '@aws-cdk/core';
+import { App } from 'aws-cdk-lib';
+import { SecurityHQ } from '../lib/security-hq';
 import { SecurityVpc } from '../lib/security-vpc';
 
+const app = new App();
+new SecurityHQ(app, 'security-hq', {
```

Comment:
In addition to the changes to `cdk.json`, `package.json` and `cloudFormationStackName` below, we can now do:

```sh
yarn diff security-hq
```

Locally to see how the local CDK definition differs from the template of the deployed stack. This is helpful for debugging.